test/test_transforms.py

```diff
@@ -229,10 +229,10 @@ def _test_librosa_consistency_helper(n_fft, hop_length, power, n_mels, n_mfcc, s
             # test mel spectrogram
             melspect_transform = torchaudio.transforms.MelSpectrogram(
                 sample_rate=sample_rate, window_fn=torch.hann_window,
-                hop_length=hop_length, n_mels=n_mels, n_fft=n_fft)
+                hop_length=hop_length, n_mels=n_mels, n_fft=n_fft, normalized=mel_norm == "slaney")
```

Comment:
From [comment](https://github.com/pytorch/audio/issues/287#issuecomment-582363778), there seems to be more than one way to normalize, is that the case? If so, wouldn't it be cleaner to just say `normalized="slaney"`?

However, if we go this direction, I second @cpuhrsch, it may be valuable to have a `SlaneyScale` transform like the `MelScale` transform that torchaudio has instead. Thoughts?

Reply:
Sorry for the confusion; the comments ("htk" vs "slaney") was about the mel-hz conversion methods, while this PR addresses an orthogonal option where the height of the triangular filters can be adjusted to have the same area. [(figure)](https://user-images.githubusercontent.com/266841/65094873-abe6b200-d973-11e9-9464-fd652f3a5faa.png)

This (and librosa's API) is somewhat confusing because there are the Mel scale from Slaney and Slaney normalization, both referring to [the auditory toolbox](https://engineering.purdue.edu/~malcolm/interval/1998-010/AuditoryToolboxTechReport.pdf) by Malcolm Slaney.

So for this specific issue I'd suggest using `normalized` or `area_normalize` for the names.

----
**NOT related to the PR but regarding the Mel scale options**:

We can later consider supporting other Mel scales (which is good to have for better librosa compliance but not my top priority).

Even in that case, having `SlaneyScale` and `MelScale` would be quite confusing; both are Mel scales, and there is no "standard" Mel scale used in literature (which is quite unfortunate). Someone uses this:

![image](https://user-images.githubusercontent.com/266841/73965194-862d5b80-48c8-11ea-95a4-fa10648f9e82.png)

which is neither Slaney nor HTK... but this scale seems to be a pretty niche one.